source/libraries/HelperFunctions.ts

```diff
@@ -47,30 +41,10 @@ export function stringTo32ByteHex(stringToEncode: string): string {
     return padAndHexlify(stringToEncode, 64, "right");
 }
 
-export async function generateTestAccounts(secretKeys: string[]): Promise<TestAccount[]> {
-    let testAccounts: Promise<TestAccount>[] = [];
-    for (let secretKey in secretKeys) {
-        const hexlifiedSecretKey = padAndHexlify(secretKeys[secretKey], 64);
-        let testAccount = await EthjsAccount.privateToAccount(hexlifiedSecretKey);
-        for (let testAccountData in testAccount) {
-            testAccount[testAccountData] = testAccount[testAccountData].toLowerCase();
-        }
-        testAccounts.push(testAccount);
-    }
-
-    return await Promise.all(testAccounts);
-}
-
-export async function initializeTestRpcClientOptions(ethHttpProviderPort: number, secretKeys: string[]): Promise<RpcClientOptions> {
-    // Generate test accounts
-    const testAccounts = await generateTestAccounts(secretKeys);
-
+export function initializeTestRpcClientOptions(ethHttpProviderPort: number, privateKey: string): RpcClientOptions {
```

Comment:
It looks like `ethHttpProviderPort` is not being used, so I think this parameter can be removed.